app/models/tigerdata_schema.rb

```diff
@@ -1,162 +1,22 @@
 # frozen_string_literal: true
-
-# Defines the Tigerdata schema in MediaFlux along with the
-# document types allowed. At this point we only support "projects"
 class TigerdataSchema
-  attr_accessor :schema_name, :schema_description
-
   SCHEMA_VERSION = "0.6.1"
 
-  def initialize(schema_name: nil, schema_description: nil, session_id:)
-    @schema_name = schema_name || "tigerdata"
-    @schema_description = schema_description || "TigerData Metadata"
-    @session_id = session_id
-  end
-
-  def create
-    define_schema_namespace
-    define_project
-  end
-
-  def define_schema_namespace
-    schema_request = Mediaflux::SchemaCreateRequest.new(name: @schema_name,
-                                                        description: @schema_description,
-                                                        session_token: @session_id)
-    schema_request.resolve
+  def initialize(namespace: "tigerdata", type: "tigerdata:project")
+    @namespace = namespace
+    @type = type
   end
 
-  def define_project
-    fields_request = Mediaflux::SchemaFieldsCreateRequest.new(
-      schema_name: @schema_name,
-      document: "project",
-      description: "Project Metadata",
-      fields: project_schema_fields,
-      session_token: @session_id
-    )
-    fields_request.resolve
-    if fields_request.error?
-      raise "Could not create or update schema #{fields_request.response_error}"
+  def fields
+    @fields ||= begin
+      logon_request = Mediaflux::LogonRequest.new
+      schema_request = Mediaflux::SchemaFetchRequest.new(session_token: logon_request.session_token, namespace: @namespace, type: @type)
```

Comment:
This fetches the field definition from Mediaflux rather than relying on our hard-coded version in Ruby (which is being deleted below, see lines 42-150)